src/subscript/config_jobs/WELLTEST_DPDS

```diff
@@ -4,11 +4,12 @@ DEFAULT  <PHASE> OIL
 DEFAULT  <OUTPUTDIRECTORY> .
 DEFAULT  <OUTFILESSUFFIX> ""
 DEFAULT  <BUILDUP_NR> 1
+DEFAULT  <GENOBS_RESULTFILE> None
```

Comment:
Unless proven in test, I don't believe it works! ;)
(ert bites)

Reply:
Have some faith! It has been proven to work in tests! The reason for it working:
 if genobs_resultf == "None":
        genobs_resultf = None
Not possibly the most elegant solution, alternatively the script could check for empty string.

Reply:
Proven manually you mean?

I saw the explicit None handling in the source, but I have been bitten before..